From 38f1e2f0ba805dfe077df9d4f5d0f7176987f058 Mon Sep 17 00:00:00 2001
From: "Nguyen Van Tu [FGA.AIS]" <TuNV16@fsoft.com.vn>
Date: Thu, 27 Sep 2018 14:49:56 +0700
Subject: [PATCH 233/641] ARM: debug-ll: Fix EARLY_PRINTK configuration add
 default y wrongly

Fix commit 146ff426285d ("ARM: debug-ll: Add support for r8a77470")

Signed-off-by: thongsyho <thong.ho.px@rvc.renesas.com>
Signed-off-by: Nguyen Van Linh [FGA.BU13] <LinhNV25@fsoft.com.vn>
Signed-off-by: vietn <vietn@fsoft.com.vn>
Signed-off-by: Nguyen Van Tu [FGA.AIS] <TuNV16@fsoft.com.vn>
---
 arch/arm/Kconfig.debug | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/Kconfig.debug b/arch/arm/Kconfig.debug
index 7fce40a..b848900 100644
--- a/arch/arm/Kconfig.debug
+++ b/arch/arm/Kconfig.debug
@@ -1633,7 +1633,6 @@ config UNCOMPRESS_INCLUDE
 config EARLY_PRINTK
 	bool "Early printk"
 	depends on DEBUG_LL
-	default y
 	help
 	  Say Y here if you want to have an early console using the
 	  kernel low-level debugging functions. Add earlyprintk to your
-- 
2.7.4

